internal/lua_tests/fs_test.lua

```diff
@@ -39,12 +39,30 @@ function main()
     end
 
     lines, err = api.fs.getProperLines(partitionRegex, resolv)
-
     if err ~= nil or lines == nil then
         api.info.error(err)
         return false
     end
 
+    fileToBeCreatedPath = "/tmp/spito-create-file-test-24tc89t221"
```

Comment:
Any reason for this name?

Reply:
we do not provide any api for random so I used this name in order to make it unique